usr/src/test/zfs-tests/tests/functional/cli_root/zfs_receive/zfs_receive_010_pos.ksh

```diff
@@ -137,6 +137,11 @@ $TOUCH $mntpnt2/f18
 $RM $mntpnt/h17
 $RM $mntpnt2/h*
 
+# Add empty objects to $fs to exercise dmu_traverse code
+for i in `seq 1 100`; do
```

Comment:
The GNU coreutils are part of a standard OmniOS distro, but are in their own package.  From a packaging POV, the ZFS test suite if it used seq(1) would require file/gnu-coreutils in its packaging. Other distros with more minimalist trappings may chafe with such a requirement.